app/api/csv.py

```diff
@@ -158,7 +158,9 @@ def get_states_daily_csv():
             CSVColumn(label="Negative Antigen Tests (People)", model_column="negativeTestsPeopleAntigen"),
             CSVColumn(label="Total Antigen Tests", model_column="totalTestsAntigen"),
             CSVColumn(label="Positive Antigen Tests", model_column="positiveTestsAntigen"),
-            CSVColumn(label="Negative Antigen Tests", model_column="negativeTestsAntigen")])
+            CSVColumn(label="Negative Antigen Tests", model_column="negativeTestsAntigen"),
+            CSVColumn(label="posNeg", model_column=None, blank=True),
```

Comment:
This is not a visible column in the public sheet, so I'm outputting a blank column here so the number and order of columns is identical (invisible columns still take up a space).

Reply:
It was convenient to order them in this way in the sheet (posNeg and then totalTestResults) we can also change the order.
Otherwise, if it's intentionally blank and should always I'd rename it to _posNeg or something with the meaning of "hidden"

Reply:
That makes perfect sense. I think we probably shouldn't change the order since someone may be relying on it now, but I will add the _ before the name.

Reply:
Looking at this, it also looks like the old Total Test Results (which was hidden column Z) got removed, so I'll make it match the public sheet now.